platforms/web/src/i18n/config.ts

```diff
@@ -25,19 +25,24 @@ export const DEFINED_LANGUAGES: LanguageDefinition[] = [
 const initI18n = async () => {
   const enabledLanguages = import.meta.env.APP_ENABLED_LANGUAGES?.split(',') || [];
   const defaultLanguage = import.meta.env.APP_DEFAULT_LANGUAGE || 'en';
+
   const supportedLanguages = filterSupportedLanguages(DEFINED_LANGUAGES, enabledLanguages);
 
   useConfigStore.setState({ supportedLanguages });
 
   if (!supportedLanguages.some(({ code }) => code === defaultLanguage)) {
     throw new Error(`The default language is not enabled: ${defaultLanguage}`);
+  } else {
+    // Set the default language on the HTML element
+    document.documentElement.lang = defaultLanguage;
```

Comment:
`} else {` is redundant here

```suggestion
  // Set the default language on the HTML element
  document.documentElement.lang = defaultLanguage;
```

Reply:
Is this even needed with the suggested change from above? This is only needed when using the on/off listeners, right?

Reply:
Correct, this is not needed anymore, i've removed this line of code in a most recent commit.